splitio/sync/synchronizer.py

```diff
@@ -248,11 +254,11 @@ def sync_all(self):
                 if not self.synchronize_splits(None):
                     attempts -= 1
                     continue
-
+                
```

Comment:
you might want to set the editor to automatically remove whitespaces in empty lines on save to avoid these

Reply:
Sorry, will check for such macro.